### ISSUE TITLE
[windows] join room text unreadable

### ISSUE THREAD

**Issue by ghost** [Nheko-Reborn/nheko#1128]
### Describe the bug

text on the confirm join dialog is very faint and barely readable. also shows room id not name, so hard to actually confirm anything

### To Reproduce

1. search for public room
2. attempt to join
3. see confirm message

### What happened?

text on confirm screen unreadable and unclear due to using id not name

### Expected behavior

text can be read and clearly shows the room name to join

### Screenshots

![screenshot_2022-07-23_143336](https://user-images.githubusercontent.com/99516846/180622537-89af9470-c78e-42ee-8370-88fe0f01afcb.png)

### Version

0.10.0-7df260f5

### Operating system

Windows

### Installation method

Windows download

### Did you use profiles?

- [ ] Profiles used?

**Comment by LorenDB** [comment]
Do you have the dark theme set in nheko?

**Comment by portno12** [comment]
yes this is using the dark theme

**Comment by LorenDB** [comment]
Yeah, that is another problem. Could you open a new issue for that?

**Comment by deepbluev7** [comment]
I completely rewrote the dialog, I hope it is fine now on your system.

**Comment by deepbluev7** [comment]
![screenshot_2023-01-04_080249](https://user-images.githubusercontent.com/16312324/210574321-12bbf281-9ddd-46bc-b689-d2d8085ce8fc.png)

Seems to still be an issue for the leave dialog

**Comment by deepbluev7** [comment]
This is with dark theme for both Nheko and Windows, so the background colors are broken.
